pkg/dnsserver/dnscontroller/handler.go

```diff
@@ -0,0 +1,43 @@
+package dnscontroller
+
+import (
+	"net"
+	"strings"
+
+	"k8s.io/klog"
+
+	"github.com/miekg/dns"
+)
+
+func (lh Lighthouse) ServeDNS(w dns.ResponseWriter, r *dns.Msg) {
+	klog.Errorf("Serving  DNSRequest")
```

Comment:
This should be `Infof` however I don't think this message adds any value over L24.